### ISSUE TITLE
reduce color contrast on non-active splits?

### ISSUE THREAD

**Issue by yshavit** [heyuch/intellij-plugin-highlight-focused-pane#1]
Firstly, this is a great plugin — thank you for making it!

Do you know if it's possible to configure the overlay to reduce the color contrast of the inactive splits? That is, make them all look a bit "washed out"?

Right now, on dark mode, the plugin lightens the inactive splits — which on the one hand makes a lot of sense, but on the other hand actually makes them stand out a bit _more_, which makes it visually ambiguous (at least to me) whether they're the active or inactive split, especially if there are only two splits total. (With three or more, my mind figures it out by playing "one of these things is not like the other".)

If we were able to reduce the contrast of the inactive split, it would be much clearer to me that it's the one I shouldn't focus on.

I'm happy to try a PR, if you like! But I figured I'd post an Issue first, in case this is something that's super easy (and also to see if it's something you'd be interested in at all).

**Comment by heyuch** [comment]
Hi, @yshavit thanks for your suggestion. Currently, you can change the overlay color in the setting `File | Settings | Other Settings | Highlight Split Pane`, with a darker color and higher alpha value.

What do you mean "reduce the contrast of the inactive split"? Is that you want to reduce the contrast of font color in the inactive pane? this function is missing now, it's welcome that you can implement it.

I suggest adding an option in the setting, let users choose which highlight style to use.

**Comment by yshavit** [comment]
>What do you mean "reduce the contrast of the inactive split"? Is that you want to reduce the contrast of font color in the inactive pane?

Correct. I'll take a stab at it, probably later this week! Thanks for the quick reply.
